config/templates/hpc.template

```diff
@@ -6,6 +6,7 @@ docs_dir: docs/HPC
 
 nav:
   - Welcome: index.md
+  - Overview modules: module_overview.md
```

Comment:
Rename:

```suggestion
  - Available software: available_software.md
```

I think the plan was to exclude this page from the menu for now?

Reply:
Indeed, I will remove it